setup/ProjectSystemToolsSetup/AssemblyRedirects.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio;
+
+// These provide code-base binding redirects for the assemblies we provide in this VSIX. For NGEN purposes,
+// we also need to update src/appid/devenv/stub/devenv.urt.config.tt inside VS repo, which is done automatically 
+// via the Roslyn Insertion tool when it consumes artifacts\Debug\DevDivInsertionFiles\DependentAssemblyVersions.csv.
```

Comment:
>artifacts\Debug\DevDivInsertionFiles\DependentAssemblyVersions.csv [](start = 50, length = 66)

Have you verified this csv file is updated correctly?

Reply:
It would also be useful if you've done a mock insertion to know if it's updating the components and packages correctly on the VS repo

Reply:
Not yet, I couldn't find that file in the local artifacts (project-system\artifacts\Debug\VSSetup\Insertion is the closest I could find)
Am I looking in the right place?

Reply:
I'll try running this through the mock insertion pipeline then :)